samples/drivers/random/src/main.c

```diff
@@ -23,20 +23,26 @@ void main(void)
 	printf("random device is %p, name is %s\n",
 	       dev, dev->config->name);
 
+	printf("The last column is a sentinel, it always should be 0x00\n");
```

Comment:
yeah thought about that. But I was thinking that ppl most likely will run examples without first carefully reading the source code and will wonder why the last column is always 0x00. 

But I am good with either comment or printf, just let me know.